### DISCUSSION TITLE
Error in installation of h2owave in R

### DISCUSSION THREAD

**Discussion by aito123** [h2oai/wave#1507] [General]
Hello. Is of my knowledge that h2owave have support also for R language, i also read the post of v.0.22.0 mentioning R support:
https://wave.h2o.ai/blog/release-0.22

Nevertheless, the examples provided start with `library(h2owave)` which assumes you have already installed the package in your computer. When i proceed to `install.packages("h2owave")` in R it appears the following error: 

```
> install.packages("h2owave")
Installing package into ‘C:/Users/Lenovo/AppData/Local/R/win-library/4.2’
(as ‘lib’ is unspecified)
Warning in install.packages :
  package ‘h2owave’ is not available for this version of R

A version of this package for your version of R might be available elsewhere,
see the ideas at
https://cran.r-project.org/doc/manuals/r-patched/R-admin.html#Installing-packages
Warning in install.packages :
  unable to access index for repository http://www.stats.ox.ac.uk/pub/RWin/bin/windows/contrib/4.2:
  cannot open URL 'http://www.stats.ox.ac.uk/pub/RWin/bin/windows/contrib/4.2/PACKAGES'
```

I guess h2owave isn't in CRAN or isn't available for r version 4.20. Is that right? Is there a solution for this problem?

Let me know,
Regards,
Santiago

**Comment by ashrith** [comment]
Hi @aito123 you are right. We are in the process of publishing this on CRAN. You can currently access the package from here [h2owave_0.22.0_R.tar.gz](https://github.com/h2oai/wave/releases/download/v0.22.0/h2owave_0.22.0_R.tar.gz) Please use, `R CMD INSTALL h2owave_0.22.0_R.tar.gz`. Also please ensure that you have the dependencies installed as follows 
`install.packages(c("jsonlite","devtools","httr","stringr","httpuv","R6"))`  I apologize for the inconvenience.

**Comment by aito123** [comment]
I suggest if the package is not available through CRAN, you could allow the download directly through Github:

Example:

`devtools::install_github("h2oai/wave")`

or

`pacman::p_load_gh("h2oai/wave")`

Using R CMD could be kinda tricky. Let  me know.

**Reply by ashrith** [reply]
@aito123 Thank you! that is wonderful advice and it makes perfect sense to not inconvenience people. I will look at this soon and revert back to you when it is ready! Thank you for your patience.

**Comment by ashrith** [comment]
@aito123 posting it here again. let me know if this works. 
```
devtools::install_github(repo="h2oai/wave@v0.22.0",subdir="r")
```
